### PR TITLE
REACH-163 Widget for boolean in customizer

### PR DESCRIPTION
**Pull request by pbidenko**
Widget for boolean node.

### CODE REVIEW

File: app/app.html [hunk 1]
```diff
@@ -488,6 +488,26 @@
         <% }); %>
     </ul>
 
+    <div class="dropdown keep-open node-settings" style="position:absolute">
```
Comment (pbidenko) [inline app/app.html]: Added "Name" field and "Hide in customizer" checkbox for boolean node.

File: app/customizer.html [hunk 2]
```diff
@@ -297,6 +297,23 @@
 
       </script>
 
+      <script type="text/template" id="widget-boolean-template">
```
Comment (pbidenko) [inline app/customizer.html]: Template for boolean node widget.

File: app/scripts/views/customizer/Widgets/Boolean.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+define(['backbone', 'BaseWidgetView'], function(Backbone, BaseWidgetView) {
```
Comment (pbidenko) [inline app/scripts/views/customizer/Widgets/Boolean.js]: Widget for boolean node.

**Comment by pboyer** [comment]
Thanks @pbidenko!  Looks great!

**Pull request merged**